src/main/resources/permissions/admin-user-permissions.csv

```diff
@@ -0,0 +1,2 @@
+users.collection.get
```

Comment:
let's have only this permission for now `ui-users.editperms`

Reply:
fixed